doc/beamline_completion.md

```diff
@@ -141,12 +150,13 @@ Effort based on above consideration. **Note when comparing with Workspace 1.0 es
 
 | Activity  | Lower Estimate (Weeks) | Upper Estimate (Weeks) |  Notes |
 | ------------- | ------------- | ------------- |------------- |
-| 1  | 2  | 8 | Mantid already provides a good template for this. Work may be rapid |
-| 2  | 4 | 12 | Looking at xarray implementation may help |
-| 3  | 2 | 12 |  |
+| 1  | 2  | 8 | ComponentInfo done in Mantid already. Shape support would be new. |
+| 2  | 1 | 8 | Looking at xarray implementation may help |
```

Comment:
What xarray implementation relating to geometry information are you referring to?

Reply:
This was just a general comment about saving information from xarray to disk. Doesn't relate to geometry specifically

Reply:
Then the effort should be covered by the generic items for saving in the main design document. The relevant part here should only take into account time for saving individual items that are specific to geometry, such as a shape.

Can you reduce the estimate accordingly?